src/renderer/widgets/SendAssetModal/ui/SendAssetModal.tsx

```diff
@@ -30,39 +30,38 @@ type Props = {
 export const SendAssetModal = ({ chain, asset, onClose }: Props) => {
   const { t } = useI18n();
   const { getBalance } = useBalance();
-  const { getTransactionFee } = useTransaction();
+  const { getTransactionFee, setTxs, txs, setWrappers, wrapTx } = useTransaction();
   const { connections } = useNetworkContext();
 
   const [isModalOpen, toggleIsModalOpen] = useToggle(true);
   const [activeStep, setActiveStep] = useState<Step>(Step.INIT);
   const [account, setAccount] = useState<Account | MultisigAccount>({} as Account);
   const [signatory, setSignatory] = useState<Account>();
   const [description, setDescription] = useState('');
-  const [transferTx, setTransferTx] = useState<Transaction>({} as Transaction);
-  const [multisigTx, setMultisigTx] = useState<Transaction>();
   const [unsignedTx, setUnsignedTx] = useState<UnsignedTransaction>({} as UnsignedTransaction);
   const [signature, setSignature] = useState<HexString>('0x0');
 
   const connection = connections[chain.chainId];
-  const transaction = multisigTx || transferTx;
+  // const transaction = multisigTx || transferTx;
 
   const { api, assets, addressPrefix, explorers } = connection;
 
-  const onInitResult = (transferTx: Transaction, multisig?: { multisigTx: Transaction; description: string }) => {
-    setTransferTx(transferTx);
-    setMultisigTx(multisig?.multisigTx || undefined);
-    setDescription(multisig?.description || '');
+  const onInitResult = (transferTx: Transaction, description?: string) => {
+    setTxs([transferTx]);
+    setDescription(description || '');
     setActiveStep(Step.CONFIRMATION);
   };
 
+  const transaction = txs[0];
+
   const checkBalance = () =>
     validateBalance({
       api,
       transaction,
       chainId: chain.chainId,
       assetId: asset.assetId.toString(),
       getBalance,
-      getTransactionFee,
```

Comment:
It really doesn't work `getTransactionFee` does it?

Reply:
Yeah, cause we need to pass transaction object